int32 ActuatorID
---
float64 VOLTAGE
float64 CURRENT_SCALE
float64 VELOCITY_SCALE
float64 ACTUATOR_TEMPERATURE
float64 INVERTER_TEMPERATURE
